addons/account/static/src/components/autosave_many2many_tax_tags/autosave_many2many_tax_tags.js

```diff
@@ -1,8 +1,11 @@
+/** @odoo-module **/
```

Comment:
You should not add it, it's not mandatory in master and was removed just recently (https://github.com/odoo/odoo/commit/6d2fda172ec9cc7642abe9625ca486c7769e8297 :smile: )